hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -336,6 +336,69 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "SECRET.DATA.SCRAMBLE",
+      desc: "Secret partitions are scrambled with a full-round PRESENT cipher."
+    }
+    { name: "PART.DATA.DIGEST",
+      desc: "Integrity of buffered partitions is ensured via a 64bit digest."
+    }
+    { name: "DAI.FSM.SPARSE",
+      desc: "The direct access interface FSM is sparsely encoded."
+    }
+    { name: "LCI.FSM.SPARSE",
+      desc: "The life cycle interface FSM is sparsely encoded."
+    }
+    { name: "KDI.FSM.SPARSE",
+      desc: "The key derivation interface FSM is sparsely encoded."
+    }
+    { name: "SCRMBL.FSM.SPARSE",
+      desc: "The scramble datapath FSM is sparsely encoded."
+    }
+    { name: "TIMER.FSM.SPARSE",
+      desc: "The background check timer FSMs are sparsely encoded."
+    }
+    { name: "DAI.CNT.REDUN",
+      desc: "The direct access interface address counter is duplicated."
+    }
+    { name: "KDI_SEED.CNT.REDUN",
+      desc: "The key derivation interface counter is duplicated."
+    }
+    { name: "KDI_ENTROPY.CNT.REDUN",
+      desc: "The key derivation entropy counter is duplicated."
+    }
+    { name: "LCI.CNT.REDUN",
+      desc: "The life cycle interface address counter is duplicated."
+    }
+    { name: "PART.CNT.REDUN",
+      desc: "The address counter of buffered partitions is duplicated."
+    }
+    { name: "SCRMBL.CNT.REDUN",
+      desc: "The srambling datapath counter is duplicated."
+    }
+    { name: "TIMER_INTEG.CNT.REDUN",
+      desc: "The background integrity check timer is duplicated."
+    }
+    { name: "TIMER_CNSTY.CNT.REDUN",
+      desc: "The background consistency check timer is duplicated."
+    }
+    { name: "TIMER.LFSR.REDUN",
+      desc: "The background check LFSR is duplicated."
+    }
+    { name: "FSM.ESCALATION",
+      desc: "All redundantly encoded FSMs are moved into an invalid state upon escalation."
+    }
+    { name: "FSM.LOCAL_ESC",
+      desc: "All redundantly encoded FSMs are moved into an invalid state upon local escalation."
+    }
```

Comment:
I am not sure if we should list the `LOCAL_ESC` CM separately for every FSM as we do for `SPARSE` (and similar for `ESCALATION` above). It may be some work yes and the list would become more verbose but it would be more consistent and rigorous. I think there is a higher risk to actually miss protecting a single FSM if all of them are combined in one item here.

Reply:
Yeah I was torn between brevity and being explicit also. Maybe we should just follow through as you say and try to stay consistent. I.e., if we list other countermeasures on a per-instance basis within this block, we should do it for all countermeasures within this block.

Across different blocks, I believe the idea was to leave this decision up to the designers (either group or collapse them).

Reply:
This approach sounds good to me.